src/options/options.html

```diff
@@ -161,7 +161,7 @@ <h1 data-i18n="__MSG_titleBehaviour__">Behaviour</h1>
 				</ul>
 			</section>
 
-			<section>
+			<section id="browser">
```

Comment:
Huh why this ID?

Reply:
See comment above. This is used to disable the Omnibox options in TB, as they are of course only applicable to browsers.